tests/test_create_employee.py

```diff
@@ -50,5 +48,51 @@ def test_create_employee_no_creds(self):
         browser.find_element(By.ID, "btnSave").click()  # Save button
 
         browser.find_element(By.LINK_TEXT, "PIM").click()
-        pass
 
+    def test_validate_employee_list_sorting_rules(self):
+        browser = self.browser
+        wait = WebDriverWait(browser, 5)
+
+        # Step 1: Open Employee List page
+        browser.find_element(By.ID, "menu_pim_viewEmployeeList").click()
+        wait.until(expected_conditions.url_contains("/pim/viewEmployeeList"))
+        self.assertEqual("Employee List", browser.find_element(By.CLASS_NAME, "selected").text)
+
+        # Step 2: Scroll down to the Employee list table
+        browser.find_element(By.ID, 'footer').location_once_scrolled_into_view
+
+        # Step 3. Click the "First (Middle) Name" table header
+        browser.find_element(By.LINK_TEXT, "First (& Middle) Name").click()
+
+        # 4. Wait for the sort to complete (hint: url)
+        wait.until(expected_conditions.url_contains("ASC"))
+        url_partial = browser.current_url.split("?")
+        self.assertEqual("sortField=firstMiddleName&sortOrder=ASC", url_partial[1])
+
+        # 5. Ensure the records on the first page listed in alphabetical order by name
+        rows = browser.find_elements(By.XPATH, "//tbody/tr")
+        list_of_names = []
+
+        for single_row in rows:
+            list_of_names.append(single_row.find_element(By.XPATH, ".//td[3]").text)
+
```

Comment:
What about names that start with a lowercase letter
example:  Alex vs bob    or Beth vs james

Reply:
Fixed. Added .upper() to convert all the names.